articles/how-to-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/history-of-nodejs/
+title: How to build a static site with GatsbyJs
+description: Node.js is a runtime server environment that uses JavaScript on the server side and asynchronous programming. It is a free and open source technology that runs on various platforms (Mac OS X, Unix, Windows, etc.)
+author: jethro-magaji
+date: 2020-08-25T00:00:00-08:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/history-of-nodejs/hero.jpg
+    alt: hero example image computer Node.js
+---
+When I first heard about GatsbyJs I was pretty confused and related it to the famous ***‘J Gatsby’*** of the movie [“The Great Gatsby”](https://www.imdb.com/title/tt1343092/), my goal for this article is not to confuse you but to get you started in building your very first static site using GatsbyJs.
+<!--more-->
+A paradigm shift has begun in the web development world, where static sites are now becoming a trend over good old dynamic sites. Building a dynamic site that require you to worry about setting up a server and/or maintain it is now obsolete.
+
+and it's not an issue when building static sites because cloud service companies like [Amazon](https://aws.amazon.com/), [Google](https://cloud.google.com/), [Netlify](https://www.netlify.com/), [Heruko](https://www.heroku.com/), [Gaysby Cloud](https://www.gatsbyjs.com/cloud/) etc. think you shouldn’t be doing so and they want software development to be fast, easy, scalable and fun not only for the developers but also for the clients because slow dynamic sites can cause high page bounce rate from users, so that’s where GatsbyJs comes in; a powerful and easy static site generator.
```

Comment:
A lot of great information here - but it's in a form of run-on sentences. Let's reorganize this info into digestible chunks

Reply:
Thanks for the recommendations.
I just corrected it.